emcc/post.js

```diff
@@ -0,0 +1,33 @@
+Module['parseAPIBlueprint'] = function(text) {
+
+    if (false === this.ready) {
+        return {err: -1, result: null};
+    }
+
+    chptr = _malloc(4);
+    buffer = _malloc(text.length+1);
+    writeStringToMemory(text, buffer);
+    res = _drafter_c_parse(buffer, 0, chptr);
+    _free(buffer);
+    ptrstr = getValue(chptr, '*');
+    parse = Pointer_stringify(ptrstr);
+    _free(ptrstr);
+    _free(chptr);
+    return {err: res, result: parse};
+};
+
+Module['validateAPIBlueprint'] = function(text) {
+
+    if (false === this.ready) {
+        return -1;
+    }
+
+    buffer = _malloc(text.length+1);
+    writeStringToMemory(text, buffer);
+    res = _drafter_c_parse(buffer, 0, 0);
+    _free(buffer);
+    return res ;
+};
+
+var parseAPIBlueprint = Module['parseAPIBlueprint'];
+var validateAPIBlueprint = Module['validateAPIBlueprint'];
```

Comment:
Can we update these to mimic the API used by Drafter and Protagonist? For example:

```js
var drafter = require('drafter.js');

var result;
var options = {
  exportSourcemap: true,
  type: 'ast'
}

try {
  result = drafter.parse('# My Blueprint', options);
} catch (err) {
  // ...
}
```